pkgs/development/python-modules/black/default.nix

```diff
@@ -36,6 +40,11 @@ buildPythonPackage rec {
   preCheck = ''
     export PATH="$PATH:$out/bin"
 
+    # Work around https://github.com/psf/black/issues/2105
+    if [[ "${lib.boolToString(stdenv.isDarwin)}" == "true" ]]; then
```

Comment:
```suggestion
    if [[ "${lib.boolToString stdenv.isDarwin}" == "true" ]]; then
```

Reply:
actually, you should just use `++ lib.optionalString stdenv.isDarwin`.

Reply:
Done.